scripts/StopTomcat.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+set -e
+
+BASEDIR=$(dirname $0)
+if [ -f $BASEDIR/../Config.sh ]; then
+  source $BASEDIR/../Config.sh
+else
+  source $HOOT_HOME/conf/Config.sh
+fi
+
+echo Stopping Tomcat...
+
+export CATALINA_BASE=$TOMCAT6_HOME
+export CATALINA_HOME=$TOMCAT6_HOME
+
+echo Tomcat home: $TOMCAT6_HOME
+
+[ -d $TOMCAT6_HOME/webapps ] || (echo Please set TOMCAT6_HOME; exit -1)
+  
+$TOMCAT6_HOME/bin/shutdown.sh || true
+sleep 10
+# If the stop didn't work, kill it hard.
+echo Forcing Tomcat shutdown...
+pkill -9 -f $TOMCAT6_HOME"/endorsed" || true 
+
+echo Tomcat stopped.
+
```

Comment:
These scripts can straight from the selma nightly test environment.  They're used to deploy the services and ui code to tomcat just before the ui cucumber tests are run.  I was unable to get this sequence of events to work properly when using the init.d scripts ('sudo service tomcat6 start', etc.)  The service would hang unexpectedly at times.  But that's not to say the whole thing couldn't be made to work with the tomcat init.d scripts if I spent more time on it.  @jasonsurratt Do you think there would be other potential problems on selma with swapping out these scripts for the tomcat service comands?  I was trying to keep the tomcat logic the same between vagrant and non-vagrant environments.